pubsub/subscriptions/add_users.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package subscription
+
+// [START pubsub_set_subscription_policy]
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/iam"
+	"cloud.google.com/go/pubsub"
+)
+
+// addUsers adds all IAM users to a subscription
+func addUsers(c *pubsub.Client, subName string) error {
```

Comment:
Please document args (other samples, too).

Reply:
Do I need to make an example for *pubsub.Topic? It requires a bit of setup

Reply:
Sorry, missed this comment. I'm on the fence. I think it could be good to show, that way users don't have to flip back and forth to see how to get a `*pubsub.Topic`. Made a similar comment in the latest review.